src/abagnale.js

```diff
@@ -3,6 +3,20 @@ import slug from 'slug';
 
 slug.defaults.mode = 'rfc3986';
 
+// A safe way to slugify values. If the input is null, undefined, or
+// some other error happens downstream, we simply return `unknown`.
+function safeSlug(value) {
+  let sluggified;
+
+  try {
+    sluggified = slug(value);
+  } catch (err) {
+    sluggified = 'unknown';
```

Comment:
It shows up as a component of the ID and is unique (see this PR description). Example: `array.unknown-2.member-value.string`